src/Button/Button.stories.tsx

```diff
@@ -63,6 +63,14 @@ export const invisibleButton = (args: ButtonProps) => {
   )
 }
 
+export const OutlineButton = (args: ButtonProps) => {
```

Comment:
I'd probably recommend this be camel case (ie. outlineButton) so that it's consistent with other stories.

Reply:
Sure, updated.